README.md

```diff
@@ -111,87 +116,42 @@ workon cfgov-refresh
 cd cfgov-refresh
 
 # Index the latest content from the API output from a WordPress and Django back-end.
-# **This requires the constants in INSTALL - Configuration to be set**
+# **This requires the constants in INSTALL - Stand alone installation to be set**
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2098/commits/2ecd3189c9096eda37a5ff143c907759e91cf1a4